package.json

```diff
@@ -1,14 +1,14 @@
 {
   "name": "puppeteer",
-  "version": "2.1.1-post",
+  "version": "3.0.0",
   "description": "A high-level API to control headless Chrome over the DevTools Protocol",
   "main": "index.js",
   "repository": "github:puppeteer/puppeteer",
   "engines": {
     "node": ">=10.18.1"
   },
   "puppeteer": {
-    "chromium_revision": "722234",
```

Comment:
Please do this change (+ any test expectation updates or other Puppeteer changes needed to support the new Chromium) in a separate CL.

The Chromium roll and the Puppeteer version bump should be separate commits.